components/css/content/_contentPage.scss

```diff
@@ -53,40 +53,119 @@
 					margin-bottom:20px;
 				}
 				ul, ol {
-					list-style-type: none;
 					li{
-						padding:3px 0;
+						padding: 3px 0;
 					}
 				}
-				ol > li, ul > li {
+				li > ol {
+					padding-top: 0;
+					padding-bottom: 0;
+				}
+				ol > li {
 					&::before {
 						width: 1em;
 						margin-left: -1em;
-						position: relative;
-						color: $color-campusGold;
-						font-size: calc-toRem(18);
 					}
 				}
+
+				// spacing for all ol li::before
 				ul > li {
 					&::before {
-						content: "•";
+						content: "";
+						width: calc-toRem(8);
+						height: calc-toRem(8);
 						display: inline-block;
 						font-size: calc-toRem(33);
-						line-height: 0;
-						top: 5px;
-						right: -6px;
+						position: relative;
+						top: calc-toRem(-2);
+						right: calc-toRem(13);
 					}
 				}
-				ol {
-					counter-reset: li;
+				// define the styles for the different levels
+				ul {
+					padding-left: 22px;
+					list-style-type: none;
+
+					> li {
+						&::before {
+							background-color: $color-campusGold;
+							border-radius: 50%;
+						}
+					}
+
+					ul {
+						> li {
+							&::before {
+								background-color: #fff;
+								border-radius: 50%;
+								border: calc-toRem(2) solid $color-campusGold;
+							}
+						}
+
+						ul {
+							> li {
+								&::before {
+									border: 0;
+									border-radius: 0;
+									background-color: $color-campusGold;
+									width: calc-toRem(8);
+									height: calc-toRem(8);
+								}
+							}
+
+						}
+					}
 				}
-				ol > li {
-					&::before {
-						counter-increment: li;
-						content: counter(li)".";
-						right: .6em;
+
+				// if the browser supports css counters, use gold bullets and numbers
+				@supports( counter-reset: li ) {
+
+					// spacing for all ol li::before
+					ol > li {
+						&::before {
+							position: relative;
+							color: $color-campusGold;
```

Comment:
Just realized that since this needs to be readable, this should be `$color-headerGold` since the campus gold doesn't provide enough contrast on white.